### ISSUE TITLE
Improve backend logging

### ISSUE THREAD

**Issue by daemontus** [sybila/biodivine-boolean-models-website#31]

**Comment by AurumTheEnd** [comment]
What did you mean by this? I see some sort of app.use(morgan('combined')); logging middleware, but am unsure of what's missing.

**Comment by daemontus** [comment]
To be honest, I'm not sure about this one :D 

I agree that basic logging is there. I might not be recalling this correctly, but I think there were one thing that might not be completely relevant anymore.

Logging server errors. I believe at some point during development, there was a "try catch" block that would just silently throw away any error and turn it into a generic HTTP 500 response. Returning 500 is ok, but the problem was that we were not logging the root cause of the error anywhere. So I guess that's one possible thing to audit.

**Comment by AurumTheEnd** [comment]
Yes, currently errors do indeed get logged as the 5XX HTTP response to the original request with the morgan logger middleware, but their root cause/stack trace isn't. Therefore this issue is still relevant.
